apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseDao.kt

```diff
@@ -75,7 +96,7 @@ private fun ResultSet.asDoedshendelse(): Doedshendelse =
         relasjon = getString("relasjon").let { relasjon -> Relasjon.valueOf(relasjon) },
         opprettet = getTidspunkt("opprettet"),
         endret = getTidspunkt("endret"),
-        status = DoedshendelseStatus.valueOf(getString("status")),
+        status = Status.valueOf(getString("status")),
```

Comment:
oh alltid en debatt om naming der, blir fort utydelig om det brukes utenfor vårt scope

Reply:
Jeg beholder den som den er for nå. Tviler på at dette blir brukt utenfor våre tjenester her.